### PR TITLE
update readme for common-host-utils

### PR DESCRIPTION
**Pull request by raunakkumar**

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+# Building the common-host-utils
+
+The cleanest way to build in this package is to do the following:
+
+1. Create an empty directory for your go workspace
+   * ```➜  mkdir godemo &&  cd godemo```
+2. Set your GOPATH to this directory
+   * ```➜  export GOPATH=`pwd` ```
+3. Get the repository using git or 'go get' (examples below)
```
Comment (rgcostea) [inline BUILDING.md]: Should we add a step to fork here?

File: BUILDING.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+   * Use git to obtain the repository
+     * ```➜  git clone https://github.com/<username>/common-host-utils.git src/github.com/hpe-storage/common-host-utils```
+   * Use 'go get' to obtain chapid
+     * ```➜  go get -d github.com/hpe-storage/common-host-utils/cmd/chapid```
+4. Change your working directory to the root of the repository
+   * ```➜  cd src/github.com/hpe-storage/common-host-utils```
+5. The tests are configured to run on linux/64, so set your GO OS
+   * ```➜  export GOOS=darwin```
```
Comment (rgcostea) [inline BUILDING.md]: should this be linux?

File: BUILDING.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+6. Build then entire repository to make sure everything compiles and tests
```
Comment (rgcostea) [inline BUILDING.md]: Build the

File: cmd/dockervolumed/managedplugin/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+```markdown
+
+ll /etc/hpe-storage/
+total 16
+-r-------- 1 root root 1159 Aug  2 00:20 container_provider_host.cert
+-r-------- 1 root root 1671 Aug  2 00:20 container_provider_host.key
+-r-------- 1 root root 1521 Aug  2 00:20 container_provider_server.cert
+```
+
+b. Additionally there is config file `volume-driver.json` present at the same location. This file can be edited
```
Comment (rgcostea) [inline cmd/dockervolumed/managedplugin/FAQ.md]: there is a config file

File: BUILDING.md (hunk 2 above)
Comment (raunakkumar) [reply BUILDING.md]: If you are running on Mac then set to GOOS = darwin or should we completely skip this?

File: BUILDING.md (hunk 1 above)
Comment (raunakkumar) [reply BUILDING.md]: will do

File: BUILDING.md (hunk 2 above)
Comment (rgcostea) [reply BUILDING.md]: I think we can skip it.

**Pull request merged**